src/routes/LandingPage.tsx

```diff
@@ -12,27 +12,106 @@ import { DatasetEntry, LocationState, ProjectEntry } from "../types";
 import { PageRoutes } from "./index";
 
 import Collection from "../colorizer/Collection";
-import { AppThemeContext } from "../components/AppStyle";
 import HelpDropdown from "../components/Dropdowns/HelpDropdown";
 import Header from "../components/Header";
 import LoadDatasetButton from "../components/LoadDatasetButton";
 import { landingPageContent } from "./LandingPageContent";
 
+const Banner = styled(FlexColumnAlignCenter)`
+  position: relative;
+  --container-padding-x: 20px;
+  padding: 30px var(--container-padding-x);
+  overflow: hidden;
+  margin: 0;
+`;
+
+const BannerTextContainer = styled(FlexColumn)`
+  --padding-x: 30px;
+  padding: var(--padding-x);
+  max-width: calc(1060px - 2 * var(--padding-x));
+
+  --total-padding-x: calc(2 * var(--padding-x) + 2 * var(--container-padding-x));
+  width: calc(90vw - var(--total-padding-x));
```

Comment:
Just curious: why this width calculation? It looks like the effect of this is that, as the user resizes the screen down past the max width of the banner text box, the x margin between the edge of the screen and the edge of the text box gradually shrinks - is that intended?

Reply:
Yup! This is mostly based off a similar calculation I'm doing for the rest of the page content.